donkeycar/pipeline/augmentations.py

```diff
@@ -117,5 +117,7 @@ def create(cls, aug_type: str, config: Config) -> iaa.meta.Augmenter:
 
     # Parts interface
     def run(self, img_arr):
+        if img_arr is None:
```

Comment:
This is a good change. This is orthogonal to the changes I made to the cameras

Reply:
I think we don't need this any longer now, b/c you fixed the camera returning `None`.

Reply:
Removed through rebasing